content/docs/user-guide/contributing/docs.md

```diff
@@ -59,6 +59,15 @@ formatted and linted as well, which is also ensured by the full setup below.
 Make sure you have a recent LTS version of [Node.js](https://nodejs.org/en/)
 (`>=12.0.0`, `<=15.x`), and install [Yarn](https://yarnpkg.com/):
 
+> In Windows, you may need to install [Python], [Visual Studio Build Tools], and
```

Comment:
Also why do you need Python to run `npm` and `yarn`?

Reply:
Not `npm` or `yarn` itself, but one of the (thousands of) packages require it. :)

Reply:
To run the docs engine locally? I'd be surprised if that's the case. @rogermparent can you confirm? Thanks

And if it's true we should probably mention it in general, not just for Windows.

Reply:
I didn't take the note of which package, and I don't want to test by removing everything but yes, Python was required to be installed by a package.

Reply:
[`node-gyp`](https://github.com/nodejs/node-gyp), which is used to build native addons like `sharp` (our image processor) depends on Python. No way around it, though I've never run into the situation where I don't have Python (probably most likely on Windows I'd guess).

Reply:
Interesting.

@iesahin mind mentioning Python in general then?

Reply:
I don't think there is a Linux or Mac installation without Python. Should we really mention it?

Reply:
Depends, what version of Python is needed? If it's 3.x then we should mention it as 2.x can still be the built-in one on MacOS and Linux I think.

Reply:
Seems like we should mention for all platforms after all @iesahin ? Per https://github.com/iterative/dvc.org/pull/2692#issuecomment-929581152

Reply:
Added Python 3.6+ as a general requirement like Yarn.